common/util/autoware_debug_tools/src/tf2pose.py

```diff
@@ -0,0 +1,87 @@
+#! /usr/bin/env python3
+
+# Copyright 2020 Tier IV, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import argparse
+import sys
+
+
+import rclpy
+from rclpy.node import Node
+from tf2_ros import LookupException
+from tf2_ros.buffer import Buffer
+from tf2_ros.transform_listener import TransformListener
+from geometry_msgs.msg import PoseStamped
+
+
+class Tf2PoseNode(Node):
+    def __init__(self, options):
+        super().__init__("tf2pose")
+
+        self._options = options
+        self.tf_buffer = Buffer()
+        self.tf_listener = TransformListener(self.tf_buffer, self)
+
+        self._pub_pose = self.create_publisher(PoseStamped, "pose", 1)
+        self.timer = self.create_timer((1.0 / self._options.hz), self._on_timer)
+
+    def _on_timer(self):
+        try:
+            (trans, quat) = self.tf_buffer.lookup_transform(
+                self._options.tf_from, self._options.tf_to, rclpy.time.Time())
+            time = self._tf_listener.getLatestCommonTime(
```

Comment:
TransformListener does not have getLatestCommonTime method.

Reply:
fixed!